__tests__/app.test.js

```diff
@@ -53,14 +55,43 @@ describe("Set of GET Tests", () => {
           expect(response.body).toHaveProperty("GET /api/articles");
         });
     });
-    test("should include description key for each endpoint", () => {
+    test("should match received endpoints with endpoints.json", () => {
+      const endpointsPath = "./endpoints.json";
+      const expectedEndpoints = JSON.parse(
+        fs.readFileSync(endpointsPath, "utf-8")
+      );
       return supertest(app)
         .get("/api/endpoints")
         .then((response) => {
-          const endpoints = response.body;
-          Object.keys(endpoints).forEach((endpoint) => {
-            expect(endpoints[endpoint]).toHaveProperty("description");
-          });
+          expect(response.body).toEqual(expectedEndpoints);
+        });
+    });
+  });
+  describe("GET Articles by ID", () => {
+    test("should return a Status Code: 200 for a successful request", () => {
+      return supertest(app).get("/api/articles/5").expect(200);
+    });
+    test("should return a Status Code: 404 for a non-existent article ID", () => {
+      return supertest(app).get("/api/articles/888").expect(404);
```

Comment:
I thinkk you should be checking the message that is sent too for error handlers!